agent/src/main/java/com/teamscale/jacoco/agent/PreMain.java

```diff
@@ -98,61 +98,48 @@ public static void premain(String options, Instrumentation instrumentation) thro
 	private static AgentOptions getAndApplyAgentOptions(String options, String environmentConfigId,
 														String environmentConfigFile) throws AgentOptionParseException, IOException, AgentOptionReceiveException {
 
-		DelayedLogger delayedLogger = new DelayedLogger();
 		List<String> javaAgents = CollectionUtils.filter(ManagementFactory.getRuntimeMXBean().getInputArguments(),
 				s -> s.contains("-javaagent"));
 		if (javaAgents.size() > 1) {
-			delayedLogger.warn("Using multiple java agents could interfere with coverage recording.");
+			LOGGER.warn("Using multiple java agents could interfere with coverage recording.");
 		}
 		if (!javaAgents.get(0).contains("teamscale-jacoco-agent.jar")) {
-			delayedLogger.warn("For best results consider registering the Teamscale JaCoCo Agent first.");
+			LOGGER.warn("For best results consider registering the Teamscale JaCoCo Agent first.");
 		}
 
 		TeamscaleCredentials credentials = TeamscalePropertiesUtils.parseCredentials();
 		if (credentials == null) {
-			delayedLogger.warn("Did not find a teamscale.properties file!");
+			LOGGER.warn("Did not find a teamscale.properties file!");
```

Comment:
When I run the sample-app on master with `logging-config=./tmp/teamscale-jacoco-agent/logging/logback.rolling-file.xml` the console output is
```
Picked up JAVA_TOOL_OPTIONS: -javaagent:tmp/teamscale-jacoco-agent/lib/teamscale-jacoco-agent.jar=config-file=./log-test.properties
```

on this branch it is
```
Picked up JAVA_TOOL_OPTIONS: -javaagent:tmp/teamscale-jacoco-agent/lib/teamscale-jacoco-agent.jar=config-file=./log-test.properties
16:00:35.529 [main] WARN com.teamscale.jacoco.agent.PreMain -- Did not find a teamscale.properties file!
16:00:35.531 [main] DEBUG com.teamscale.jacoco.agent.options.AgentOptionsParser -- Parsing options: config-file=./log-test.properties
```
which I would not have expected as the log output is expected to be written to the file system. Not 100% sure though where those logs are written to the console though.

Reply:
I think this was caused by the fact that the delayed appender wasn't actually used at all - I think for some reason a console logger was picked up somewhere instead? At least in my current implementation it is not actually used when calling `LOGGER.error(...)` :thinking: I'm working on it